kobo/apps/openrosa/apps/logger/exceptions.py

```diff
@@ -10,6 +10,15 @@ class BuildDbQueriesNoConfirmationProvidedError(Exception):
     pass
 
 
+class ConflictingXMLHashInstanceError(Exception):
+    pass
+
```

Comment:
We may get rid of this class and at least rename it. 

Please see @jnm's comment 

https://github.com/kobotoolbox/kobocat/pull/876#discussion_r1253934049